AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -512,12 +512,17 @@ cna.com.tw##.qrcode_box
 ||static.fastpic.ru/android.png
 itao.com###itao-bottom
 stitcher.com###appnag
+9gag.com##a[class="get-the-app-banner"]
+||9gag.com/s/*/static/dist/*/web/img/sidebar-download-app.png
```

Comment:
2 KB only. Can be just hidden.

Reply:
Also removed obsolete rules and blocked mobile annoyances
https://github.com/AdguardTeam/AdguardFilters/commit/7df3bb2bf83678e55323b9711e9a0b02107e4f90
(popup is blocked using two CSS rules because scrolling is blocked, but on iOS CSS rules cannot be used and page will be broken).